### PR TITLE
Pub 364 test

### PR DESCRIPTION
**Pull request by ShufenZ**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import io.restassured.response.Response;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.futurehearings.pip.acceptance.common.test.*;
```
Comment (Matthew-Summerbell) [inline src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java:12]: Please remove * imports

**Review by Matthew-Summerbell — CHANGES_REQUESTED** [review]
Tests look good, please remove * import and add removed Infrastructure files

File: src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java (hunk 1 above)
Comment (ShufenZ) [reply src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java]: done

File: src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/GETPublicationsTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                                            "Test",
+                                            publicationsRootContext,
+                                            null,
+                                            HttpMethod.GET);
+
+        assertEquals(HttpStatus.OK.value(), response.statusCode());
+    }
+
+    @Test
+    void testGetPublicationNotAuthorised() {
```
Comment (Matthew-Summerbell) [inline src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/GETPublicationsTest.java:41]: For constancy use americanised spelling of Authorized

File: src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                                            "Test",
+                                            publicationsRootContext,
+                                            null,
+                                            HttpMethod.POST
+        );
+        assertEquals(HttpStatus.CREATED.value(), response.statusCode());
+    }
+
+    @Test
+    void testPostPublicationNotAuthorised() {
```
Comment (Matthew-Summerbell) [inline src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java:41]: For constancy use americanised spelling of Authorized

**Review by Matthew-Summerbell — CHANGES_REQUESTED** [review]

File: src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java (hunk 3 above)
Comment (ShufenZ) [reply src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java]: Done

**Comment by ShufenZ** [comment]
@mjotichmcts Could you please merge your branch pub-364 to master and resolve the conflicts. Thanks!

**Review by mjotichmcts — APPROVED** [review]

**Review by Matthew-Summerbell — APPROVED** [review]

**Review by Matthew-Summerbell — APPROVED** [review]

**Review by mjotichmcts — APPROVED** [review]

**Pull request merged**